src/dynarec/la64/dynarec_la64_avx_0f.c

```diff
@@ -232,6 +232,46 @@ uintptr_t dynarec64_AVX_0F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, in
                 VPICKVE2GR_DU(gd, d1, 0);
             }
             break;
+        case 0x54:
+            INST_NAME("VANDPS Gx, Vx, Ex");
+            nextop = F8;
+            GETGY_empty_VYEY_xy(v0, v1, v2, 0);
+            if (vex.l) {
```

Comment:
I would suggest we have some macros like `XVAND_V_xy`, which can be defined as:

```
#define XVAND_V_xy(vd, vj, vk) do { if (vex.l) ... else ... } while 0
```

But this is up to you.

Reply:
SIMD insts number is much more then scalar insts.
Should we add all those Macros to all "simple" SIMD insts? (simple means only 1-2 insts to emit)
Or just add macro for  logic/shift/arith ops?